src/main/java/io/javalin/core/ExceptionMapper.kt

```diff
@@ -22,16 +22,19 @@ class ExceptionMapper {
 
     internal fun handle(exception: Exception, ctx: Context) {
         ctx.inExceptionHandler = true // prevent user from setting Future as result in exception handlers
-        if (noUserMapperFound(exception) && HttpResponseExceptionMapper.canHandleException(exception)) {
-            return HttpResponseExceptionMapper.handleException(exception, ctx)
-        }
         val exceptionHandler = this.getHandler(exception.javaClass)
-        if (exceptionHandler != null) {
-            exceptionHandler.handle(exception, ctx)
-        } else {
-            log.warn("Uncaught exception", exception)
-            HttpResponseExceptionMapper.handleException(InternalServerErrorResponse(), ctx)
+
+        when {
+            exceptionHandler != null ->
```

Comment:
Why newline here?

Reply:
Just trying to keep the line-length down. I'm happy to put these two cases on single lines, though.